lib/shared/components/navs/useNav.tsx

```diff
@@ -27,6 +28,14 @@ export function useNav() {
     },
   ]
 
+  // To-do: Remove this when veBAL is live
+  if (isDev || isStaging) {
+    appLinks.push({
+      href: '/vebal',
+      label: 'veBAL (wip)',
```

Comment:
There's no need to say it's a wip here.
```suggestion
      label: 'veBAL',
```

Reply:
Under this conditional we should just replace the redirect with this.

Reply:
Sure, but when do you ever need the vebal redirect in dev?

Reply:
I really don't think this is needed for other devs either. No one is going to get confused. But if you want it then no worries.